Copy: Re-use existing neg and conj kernel implementations

peterbell10 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #69026
* #68963
* #68950
* __->__ #68949

This reuses the existing `neg_kernel` and `conj_kernel`
implementations for copy, saving some binary size and compile time.

Differential Revision: [D33064390](https://our.internmc.facebook.com/intern/diff/D33064390)

peterbell10 committed:
Copy: Re-use existing neg and conj kernel implementations

This reuses the existing `neg_kernel` and `conj_kernel`
implementations for copy, saving some binary size and compile time.

[ghstack-poisoned]

peterbell10 committed:
Update on "Copy: Re-use existing neg and conj kernel implementations"


This reuses the existing `neg_kernel` and `conj_kernel`
implementations for copy, saving some binary size and compile time.

[ghstack-poisoned]

peterbell10 committed:
Update on "Copy: Re-use existing neg and conj kernel implementations"


This reuses the existing `neg_kernel` and `conj_kernel`
implementations for copy, saving some binary size and compile time.

[ghstack-poisoned]

peterbell10 committed:
Update on "Copy: Re-use existing neg and conj kernel implementations"


This reuses the existing `neg_kernel` and `conj_kernel`
implementations for copy, saving some binary size and compile time.

[ghstack-poisoned]

anjali411 reviewed (approved):
This looks great! thank you

anjali411 commented:
@anjali411 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33064390).

peterbell10 committed:
Update on "Copy: Re-use existing neg and conj kernel implementations"


This reuses the existing `neg_kernel` and `conj_kernel`
implementations for copy, saving some binary size and compile time.

Differential Revision: [D33064390](https://our.internmc.facebook.com/intern/diff/D33064390)

[ghstack-poisoned]

peterbell10 committed:
Update on "Copy: Re-use existing neg and conj kernel implementations"


This reuses the existing `neg_kernel` and `conj_kernel`
implementations for copy, saving some binary size and compile time.

Differential Revision: [D33064390](https://our.internmc.facebook.com/intern/diff/D33064390)

[ghstack-poisoned]

peterbell10 committed:
Update on "Copy: Re-use existing neg and conj kernel implementations"


This reuses the existing `neg_kernel` and `conj_kernel`
implementations for copy, saving some binary size and compile time.

Differential Revision: [D33064390](https://our.internmc.facebook.com/intern/diff/D33064390)

[ghstack-poisoned]

peterbell10 committed:
Rebase and remove as_strided x-fail on "Copy: Re-use existing neg and conj kernel implementations"


This reuses the existing `neg_kernel` and `conj_kernel`
implementations for copy, saving some binary size and compile time.

Differential Revision: [D33064390](https://our.internmc.facebook.com/intern/diff/D33064390)

[ghstack-poisoned]

peterbell10 committed:
Update on "Copy: Re-use existing neg and conj kernel implementations"


This reuses the existing `neg_kernel` and `conj_kernel`
implementations for copy, saving some binary size and compile time.

Differential Revision: [D33064390](https://our.internmc.facebook.com/intern/diff/D33064390)

[ghstack-poisoned]

anjali411 commented:
@anjali411 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33064390).

anjali411 commented:
Hey @peterbell10 can you please rebase this stack?

peterbell10 committed:
Update on "Copy: Re-use existing neg and conj kernel implementations"


This reuses the existing `neg_kernel` and `conj_kernel`
implementations for copy, saving some binary size and compile time.

Differential Revision: [D33064390](https://our.internmc.facebook.com/intern/diff/D33064390)

[ghstack-poisoned]

anjali411 commented:
@anjali411 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33064390).
